services/oada_upload/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.1'
 services:
   oada-upload:
     build:
-      context: ./
+      context: ../oada_upload
```

Comment:
I guess I see what your doing. I assume your trying to tie all these together so you `-f` them all together? 

Change my mind: docker-compose.yml files in a service should be a standard alone file that will bring up whatever is needed to test the service. This may be include things like `gpsfake` so `gps2tsdb.py` has something to work with. Without that, me thinks automated testing will be very hard.

Reply:
Agreed, however this is easy to change once we write an automated test. As of right now my testing procedure needs this

Reply:
Moving discussion here https://github.com/OATS-Group/isoblue-avena/pull/45#discussion_r461692316